packages/coin98-wallet/src/lib/coin98-wallet.ts

```diff
@@ -0,0 +1,167 @@
+}: Coin98WalletParams = {}): WalletModuleFactory<InjectedWallet> => {
+  return async () => {
+    const mobile = isMobile();
+    const installed = await isInstalled();
+
+    if (mobile) {
+      return null;
+    }
+
+    return {
+      id: "coin98-wallet",
+      type: "injected",
+      metadata: {
+        name: "Coin98 Wallet",
+        description: "Crypto Wallet & Defi Gateway",
```

Comment:
We are taking wallet descriptions from [awesomenear](https://awesomenear.com/). Can you change this to the one that is on there.

Reply:
commented fixed